src/Microsoft.VisualStudio.LanguageServices.Razor/DefaultTemplateEngineFactoryService.cs

```diff
@@ -11,6 +13,7 @@ namespace Microsoft.CodeAnalysis.Razor
 {
     internal class DefaultTemplateEngineFactoryService : RazorTemplateEngineFactoryService
     {
+        private const string MvcAssemblyName = "Microsoft.AspNetCore.Mvc";
```

Comment:
This is a meta package. We should be looking for `Microsoft.AspNetCore.Mvc.Razor` since that's what actually enables Razor like features.